modules/serverless-subscription-event-delivery-config.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// serverless/event_workflows/serverless-event-delivery.adoc
+
+[id="serverless-subscription-event-delivery-config_{context}"]
+= Configuring event delivery failure parameters using subscriptions
+
+Developers can configure event delivery parameters for individual subscriptions by modifying the `delivery` settings for a `Subscription` object.
+
+.Example subscription YAML
```

Comment:
this YAML examples comes a bit out of the blue and does not fit to the description of `kn subscription create` which uses `kn`. You should also use `kn` here and leverage its options `--sink-dead-letter` here, and if something is missing (ee.g. I'm not sure that retries can be configured via the CLI, we should (a) implement this asap  and (b) point out how to do it by  using/modifyng YAML documents).

Reply:
Why is this not described along in the section where you create a subscription ? It feels unnatural to have it separate from the `subscription create` section (also, the `create` section is quite slim so might benefit of having these extra options to be described there)

Reply:
It's a separate module because it's a reference module to show the YAML as opposed to part of a procedure - it's also optional, so we should keep the create procedures simple and provide this on its own like this as an optional reference for folks who want to use it.